x-pack/plugins/data_visualizer/public/application/index_data_visualizer/components/index_data_visualizer_view/index_data_visualizer_view.tsx

```diff
@@ -468,14 +468,13 @@ export const IndexDataVisualizerView: FC<IndexDataVisualizerViewProps> = (dataVi
                   />
                 </div>
               </EuiPageContentHeaderSection>
-
-              <EuiFlexGroup
-                alignItems="center"
-                justifyContent="flexEnd"
-                gutterSize="s"
-                data-test-subj="dataVisualizerTimeRangeSelectorSection"
-              >
-                {currentDataView.timeFieldName !== undefined && (
+              {currentDataView.timeFieldName !== undefined && (
```

Comment:
Testing against a data view which contains fields of type `date` but which I have configured not to use a time field, I still see the date picker:

<img width="1090" alt="image" src="https://user-images.githubusercontent.com/7405507/203576789-c60725c0-5bac-4f28-bca0-95554bdea8a7.png">

In Discover, the time picker is correctly being hidden.